server/adaptors/integrations/__data__/repository/apache/apache-1.0.0.json

```diff
@@ -5,7 +5,11 @@
     "description": "Apache web logs collector",
     "license": "Apache-2.0",
     "type": "logs_apache",
-    "author": "OpenSearch",
+    "labels": [
+      "log",
+      "communication",
+      "http"],
```

Comment:
Would appreciate if autoformatting was applied to all of the documents, just to avoid the refactoring later on

Reply:
ok